cli/cmd/testdata/inject_gettest_deployment.good.golden.yml

```diff
@@ -63,6 +63,8 @@ spec:
               fieldPath: metadata.namespace
         - name: CONDUIT_PROXY_DESTINATIONS_AUTOCOMPLETE_FQDN
           value: Kubernetes
+        - name: CONDUIT_PROXY_DEPLOYMENT
```

Comment:
these test outputs should not mention `CONDUIT_PROXY_DEPLOYMENT`?

Reply:
thanks!  I'm just failing at reading `go test` output over here... 😵